conf/metrics.conf

```diff
@@ -833,8 +833,9 @@ metric {
 
     group "phishing" {
         symbol "PHISHING" {
-            weight = 3.0;
```

Comment:
Then would you mind to share your collection by adding it to the redirectors white list?